common/flatpak-transaction.c

```diff
@@ -872,6 +1055,14 @@ flatpak_transaction_set_disable_prune (FlatpakTransaction *self,
   priv->disable_prune = disable_prune;
 }
 
+/**
+ * flatpak_transaction_set_disable_dependencies:
+ * @self: a #FlatpakTransaction
+ * @disable_dependencies: whether to disable dependencies
+ *
+ * Sets whether the transaction should ignore dependencies when resolving
+ * operations.
```

Comment:
Maybe we should mention what dependencies are (i.e. the runtime the app needs).

Reply:
Yes, I'll add that